src/scripts/create_ap

```diff
@@ -1741,10 +1748,21 @@ elif [[ $NO_DNSMASQ -eq 0 ]]; then
     if [[ "$DHCP_DNS" == "gateway" ]]; then
         DHCP_DNS="$GATEWAY"
     fi
+
+    DHCP_RANGE_LOWER=1
+    DHCP_RANGE_UPPER=254
+    if [[ $DHCP_RANGE =~ ^[0-9]+,[0-9]+$ ]]; then
+        IFS=',' read -ra DHCP_NUMS <<< "$DHCP_RANGE"
+        if ((DHCP_NUMS[0] >= 1 && DHCP_NUMS[0] <= 254 && DHCP_NUMS[1] >= 1 && DHCP_NUMS[1] <= 254 && DHCP_NUMS[0] < DHCP_NUMS[1])); then
+            DHCP_RANGE_LOWER=${DHCP_NUMS[0]}
+            DHCP_RANGE_UPPER=${DHCP_NUMS[1]}
+        fi
+    fi
```

Comment:
I see you use `LOWER` and `UPPER` here instead of `START` and `END` i mentioned above.
Feel free to change these to what i suggested or the names i suggested to these, they both have same purpose...

Reply:
Oh if you do split the option into two then this logic needs to be adopted to that case ofcourse :wink: